build.assets/build-test-compat.sh

```diff
@@ -34,15 +34,19 @@ DISTROS=(
   "debian:9"
   "debian:10"
   "debian:11"
+  "debian:12"
```

Comment:
Should we drop Debian 9 now? It hasn't been receiving security updates for over 3 years.

Reply:
Technically, it's only been not receiving security updates for a little over a year ago (LTS EOL was 2022-06-30), but happy to remove it, especially as we're already adding multiple OSes.